mkeditorpacks/mkeditorpacks

```diff
@@ -1,13 +1,38 @@
-#! /usr/bin/env python3
+#!/usr/bin/env python3
 #-*- coding: UTF-8 -*-
 
+# Copyright (c) 2014-2017, Daemon Developers
```

Comment:
Strictly speaking, you may use the dash between years only if every year was actually one where the code was touched by the named copyright holder. Though again, it's not like we cared much before.